zerver/openapi/zulip.yaml

```diff
@@ -20358,6 +20375,12 @@ components:
             Is `null` for channels with no message history.
 
             **Changes**: New in Zulip 2.1.0.
+        is_recently_active:
+          type: boolean
+          description: |
+            Whether the channel has any new messages sent to it recently.
+
+            **Changes**: New in Zulip 9.0.
```

Comment:
This needs to be updated and also record the feature level, like every other API change.

Reply:
Done